### ISSUE TITLE
Farming Station adds server lag. 

### ISSUE THREAD

**Issue by melekaiah** [SleepyTrousers/EnderIO#3986]
#### Issue Description:
Farming station seems to add quite a lot of server lag

#### What happens:
after setting up the farming station, every few seconds there is notable server lag, (every 5 to 10 seconds)

#### What you expected to happen:
no lag, 1.7.10 ran 6 tree farms in one area, no server lag, now server lag. 
lag persits after farming station removed. 
lag persits until chunk of farming station is regenerated
tested 7 times, to track issue to farming station, first dleted region files, then regened chunks, 
lag dissapears when chunk is out of load. 
no exceptions or bugs in log. just noticeably laggy. 

server is run on private IP, with 100mb uplink. quadcore processor at 3.2 ghz, and 16 gig ram, 
live world on ssd, backups on normal HDD, connection to server 50mb, 
hence im pretty sure the lag is server side, it has been confirmed by other players that the whole server starts lagging dramatically as soon as chunk with farming station has been loaded. 

#### Steps to reproduce:

1. Set up a capacitor bank (x10), set up alloy smelter (x1), set up striling generator (x18) 
2. hook up chest full of oak to input into alloy smelter, 
3. alloy smelter extracts to stirling generators only, fills them with charcoal from smelter.
4. all 18 work fine, zero server lag. power ticks every second no slow downs
5. and farming station x1, chests x3, trashcan x1, capacitor x1, 
6. add tier4 powered axe to farming station, 
7. out put oak to chest 1, leaves chest 2, and saplings chest 3, 
8. set priority of chests to 1, trash can to 0, once chests are full output will go to trash. 
9. power and start the farming station- it begins to lag. 
10. look at power chargeing or open inventories to see slow down. 
...
____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: 3.1.157
- EnderCore: 4.1.62
- Forge: 1.10.2 recommended - 2185

#### Your most recent log file where the issue was present: 

not present in log

**Comment by HenryLoenwind** [comment]
Please run the internal profiler https://www.reddit.com/r/feedthebeast/comments/5mxn51/vanilla_has_an_built_in_profiler_that_not_a_lot/ for a while. Thx.

**Comment by melekaiah** [comment]
well do, been using tickprofiler a bit, and was getting pretty high results from the farming station - at least it was the largest of all the things loaded, but i didnt record them ill do more checking over the weekend.

**Comment by HenryLoenwind** [comment]
any kind of profiling will help, it's just that seeing a general slowdown when a machine is there isn't helping us at all in finding out what may be the problem.

**Comment by choekstr** [comment]
I also have huge "lag" when I turn on my farming station and was curious if this was server or client related, farming station or mystical ag crop related.  I ran some testing on SF3 with highly accelerated crops with 9 Tier2 growth crystals and fertilized dirt with a farming station only doing 9 crops with no capacitor in farming station.  /debug showed minimal server lag while farming station was on and harvest:
TileFarmStation - 0.42%/0.24%
and no entry at all for the TileFarmStation when it was off.

I then ran sampler on the client to see if it was client side and it shows a ton of lag spikes every time the farming station harvested (which is pretty much persistent).  I turned off the farming station and manually harvested and every graphical update the essence seed/plant did it would hiccup the client.  

Conclusion and TL:DR: Farming station is not producing any server lag for me, it is client side mystical agriculture crops that produce significant FPS drops every time it redraws a model.
